.gitignore

```diff
@@ -8,9 +8,6 @@ auto-save-list
 tramp
 .\#*
 
-# npm lock file
```

Comment:
unrelated change here --

I was rusty on perspective dev, used `npm` by mistake instead of yarn and, without the "new untracked file" cue in `git status` I didn't realize I had gone astray. Ended up with a busted node_modules directory from it

Of course we also don't want package-lock.json checked in. So I am happy to undo this, just wanted to raise the issue. Maybe there is a better way to help people avoid using npm in perspective dev